plugins/rebalance/rest.go

```diff
@@ -44,14 +44,18 @@ func rebalanceStartHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	rebalinfo := createRebalanceInfo(volname, &req)
-	if rebalinfo == nil {
-		logger.WithError(err).Error("failed to create Rebalance info")
```

Comment:
Does this require to log the volume name as well?

Reply:
IMO : Its easier to debug if the resource name is provided with error/debug message